src/test/java/ContactTest.java

```diff
@@ -8,75 +9,55 @@
 
 public class ContactTest {
 
+    Contact contact;
+    @Before
+    public void initialize() {
+       contact = new Contact("Namey", "Namerson", "A Palace", "130077", "01/01/1999", "email@email.com");
+    }
     @Test
     public void testContactClassIsNotNull() {
-        Contact contact = new Contact();
         assertNotNull(contact);
     }
 
     @Test
-    public void testCanUpdateLastName() {
-        Contact contact = new Contact();
-        assertEquals(null, contact.returnLastName());
-        String input = "Sir";
-        InputStream in = new ByteArrayInputStream(input.getBytes());
-        System.setIn(in);
-        contact.updateLastName(input);
-        assertEquals("Sir", contact.returnLastName());
+    public void testCanUpdateFirstName() {
+        assertEquals("Namey", contact.returnFirstName());
```

Comment:
There is a `requireEqual` or something similar. That will prevent the test from running if the condition is not met.